### DISCUSSION TITLE
Output of Prob 4 - Task 05

### DISCUSSION THREAD

**Discussion by SMRazaRizvi96** [chulminy/CIVE497-CIVE700#165] [Q&A]
Can anyone confirm the output for the image `IMG_0080.JPG`, in Problem 4 of Task 05?

The original image is:

![IMG_0080](https://github.com/chulminy/CIVE497-CIVE700/assets/74411560/b55e009f-5bae-4215-8cec-73ab328b1b80)

And the reference output given is:

![IMG_0080](https://github.com/chulminy/CIVE497-CIVE700/assets/74411560/cc0bd7fc-c8df-4469-b361-cdcaf212564c)

However, I believe that the output should be:

![IMG_0080](https://github.com/chulminy/CIVE497-CIVE700/assets/74411560/b78ae5b9-23b0-4471-b61e-d6e094ccf1f2)

Can anyone please check this?

**Comment by chulminy** [comment]
That's okay. See this
![image](https://github.com/chulminy/CIVE497-CIVE700/assets/3227103/379c468f-5e60-4690-b0ce-f8e242869c68)
